scaling_performance/optimizing_storage.adoc

```diff
@@ -171,14 +171,57 @@ to nodes to support a healthy cluster.
 * OpenStack Cinder: OpenStack Cinder tends to be adept in ROX access mode use cases.
 * Databases: Databases (RDBMSs, NoSQL DBs, etc.) tend to perform best with dedicated block storage.
 
-[[choosing-a-docker-graph-driver]]
-== Choosing a Docker Graph Driver
+[[choosing-a-graph-driver]]
+== Choosing a Graph Driver
+
+Container runtimes store images and containers in a graph driver (a pluggable storage
+technology), such as `DeviceMapper` and`Overlay`. Each have
+advantages and disadvantages.
+
+For more information about `Overlay`, including supportability and usage
+caveats, see the
+link:https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html-single/7.3_Release_Notes/index.html#technology_previews_file_systems[Red
+Hat Enterprise Linux (RHEL) 7 Release Notes].
+
+.Graph Driver Comparisons
+|===
+|Name |Description |Benefits |Limitations
+
+|Device Mapper loop-lvm
+|Uses the device mapper thin provisioning module (dm-thin-pool) to implement copy
+on write (CoW) snapshots. For each device mapper graph location, thin pool is
+created based on two block devices, one for data and one for metadata. By
+default, these block devices are created automatically by using loopback mounts
+of automatically created sparse files.
+|It works out of the box, so it is useful for prototyping and development purposes.
+a|* Not all Portable Operating System Interface for Unix (POSIX) features work (for
+example, `O_DIRECT`). Most importantly, this mode is unsupported for production
+workloads.
+* All containers and images share the same pool of capacity. It cannot be resized
+without destroying and re-creating the pool.
+
```

Comment:
Too many _it_ references ?? 
For ex: _It cannot be resized..._ What cannot be resized?